pkg/ddc/thin/transform_fuse_test.go

```diff
@@ -392,6 +392,9 @@ func TestThinEngine_transformFuse(t1 *testing.T) {
 	}
 	dataset := &datav1alpha1.Dataset{
 		Spec: datav1alpha1.DatasetSpec{
+			PublicOptions: map[string]string{
+				"c": "d",
+			},
 			Mounts: []datav1alpha1.Mount{{
 				MountPoint: "abc",
 				Options:    map[string]string{"a": "b"},
```

Comment:
How about creating a test case that the options has {"a": "b"}, and publicOptions has {"a": "c"}? I think the expected result is  {"a": "b"},.

Reply:
ok

Reply:
Agree with @cheyang . I think the expected priority order when overriding options may be 
1. private encrypted options
2. private options
3. shared encrypted options
4. shared options